153. Find Minimum in Rotated Sorted Array.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Sat Feb  5 18:07:46 2022
+
+@author: BJay
+"""
+#Time Complexity : O(log(n))
+#Space COmplexity : O(1)
+#Runs fine on leetcode
+#Had to see 1 hint
+
+class Solution:
+    def findMin(self, nums: List[int]) -> int:
+        min = 5000
```

Comment:
Please try to implement without using `min`. [ when the constraint is not given and you will not know the initial value of `min` ].